flycheck.el

```diff
@@ -3281,8 +3320,12 @@ list."
 
 POS defaults to `point'."
   (interactive)
-  (-when-let* ((error (tabulated-list-get-id pos))
-               (buffer (flycheck-error-buffer error)))
+  (-when-let* ((error (tabulated-list-get-id pos)))
```

Comment:
I factored out the functionality from this function into `flycheck-jump-to-error` so that it could be used above.